xla/service/buffer_assignment.h

```diff
@@ -43,6 +43,8 @@ limitations under the License.
 
 namespace xla {
 
+inline constexpr int64_t kPersistentTempMemorySpaceColor = 100;
```

Comment:
Should this be next to kInvalidColor?
Why was the value 100 chosen? Are we sure that it doesn't collide with other values?

Reply:
No particular reason for chosing 100 as the number, just randomly select a value that is not collide with others. Yes, it should be better that XLA put all available ids in a centralized places, but nccl memory space id was hardcoded in the code. Can we make this as TODO that will be fixed in a separate PR?